t/17-labels_carry_over.t

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2016 SUSE Linux GmbH
```

Comment:
we agreed @aaannz will take a look into checking this automatically.

@coolo
1) wanna keep as is and merge
2) I change this single line with a very cost-inefficient commit fixup when other files are still wrong
3) you have more suggestions how to improve, e.g. regarding the code in Scheduler
?